contrib/debian/rules

```diff
@@ -18,7 +18,7 @@ override_dh_auto_clean:
 # Yea, autogen should be run on the source archive, but I like doing git archive
 override_dh_auto_configure:
 	./autogen.sh
-	./configure
+	./configure --without-miniupnpc --with-gui=qt4
```

Comment:
I need to test it on all 5 versions anyway, if something does not work out we'll adapt